prow.sh

```diff
@@ -540,7 +568,18 @@ start_cluster () {
         fi
         image="csiprow/node:latest"
     else
-        image="kindest/node:v${CSI_PROW_KUBERNETES_VERSION}"
+        local varname
+        varname="CSI_PROW_KIND_IMAGE_DIGEST_$(echo "${CSI_PROW_KUBERNETES_VERSION}" | tr . _)"
+        local digest
+        digest=$(eval echo "\$$varname")
+        if [ "$digest" ]; then
+            image="kindest/node:v${CSI_PROW_KUBERNETES_VERSION}@$digest"
+        else
+            # We shouldn't get here because of override_k8s_version, but let's keep the
```

Comment:
do we not have a canary job where we run against k8s master?

Reply:
Yes. That's covered by the preceding ` if [[ "${CSI_PROW_KUBERNETES_VERSION}" =~ ^release-|^latest$ ]]; then` which picks the locally built `csiprow/node:latest` as image.